pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -15,31 +15,52 @@
  */
 package com.linkedin.pinot.controller.api.restlet.resources;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.linkedin.pinot.common.metadata.segment.RealtimeSegmentZKMetadata;
+import com.alibaba.fastjson.JSONObject;
```

Comment:
Seems not just this file use this json library. Will clean up in the future if needed.